README.md

```diff
@@ -175,6 +175,16 @@ resource type in the cluster. Currently, the following operations are supported:
 | `get offsets [topic]` | Number of messages per partition along with start and end times |
 | `get topics` | All topics in the cluster |
 
+#### rebalance
+
+```
+topicctl rebalance [flags]
+```
+
+The `rebalance` subcommand will do a full broker rebalance for all the topic configs in a given topic prefix path.
```

Comment:
I'm assuming you just used the wording from the rebalance section but "full broker rebalance" suggests something else to me. How about "full cluster rebalance" or "all broker rebalance"? This change should be made in all places it appears.

Reply:
"full cluster rebalance" this makes sense to me..

Reply:
I am not sure about modifying at other places. But I am modifying atleast in the subcommand `rebalance`